.github/workflows/container-build-push-gea.yml

```diff
@@ -139,14 +169,14 @@ jobs:
             echo "tags<<EOF" >> $GITHUB_OUTPUT
             echo "type=ref,event=pr,prefix=${{ steps.properties.outputs.prefix }}pr-,suffix=${{ steps.properties.outputs.suffix }}" >> $GITHUB_OUTPUT
 
-            echo "type=raw,value=latest,prefix=${{ steps.properties.outputs.prefix }},suffix=${{ steps.properties.outputs.suffix }},enable=${{ steps.latest.outputs.is-latest-tag == 'true' }}" >> $GITHUB_OUTPUT
-            echo "type=raw,value=stable,prefix=${{ steps.properties.outputs.prefix }},suffix=${{ steps.properties.outputs.suffix }},enable=${{ steps.latest.outputs.is-latest-tag == 'true' }}" >> $GITHUB_OUTPUT
+            echo "type=raw,value=latest,prefix=${{ steps.properties.outputs.prefix }},suffix=${{ steps.properties.outputs.suffix }},enable=${{ steps.latest.outputs.is-latest-tag == 'true' && inputs.enable-latest == true }}" >> $GITHUB_OUTPUT
+            echo "type=raw,value=${{ inputs.stable-name }},prefix=${{ steps.properties.outputs.prefix }},suffix=${{ steps.properties.outputs.suffix }},enable=${{ steps.latest.outputs.is-latest-tag == 'true' }}" >> $GITHUB_OUTPUT
```

Comment:
it was already similar before but i somehow dont understand that whole thing - aren't these shell lines overwriting each other, so the only variables that are really valid will be those from the last command?

Reply:
not really. it's creating a string as input for https://github.com/docker/metadata-action?tab=readme-ov-file#tags-input it's some kind of template with csv values that is evaluated by the docker metadata-action for container image tags.